xml/operations-configuring-identity-websso.xml

```diff
@@ -840,4 +855,254 @@ keystone_sp_conf:
+       </para>
+       <warning>
+        <para>
+         Some OpenID Connect IdPs such as Google requires the hostname in the "redirect_uri" to be a public FQDN. In that case, the hostname in Keystone public endpoint must also be a public FQDN and must match the one specified in the "redirect_uri".
+        </para>
+       </warning>
+      </listitem>
+      <listitem>
+       <para>
+        <literal>crypto_passphrase</literal>: corresponding to <link xlink:href="https://github.com/zmartzone/mod_auth_openidc/blob/master/auth_openidc.conf">auth_openidc OIDCCryptoPassphrase</link>. If left blank, a random cryto passphrase will be generated.
+       </para>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+    <listitem>
+     <para>
+      Commit the changes to your local git repository.
+     </para>
+<screen>cd ~/openstack/ardana/ansible
+git add -A
+git commit -m "add OpenID Connect configuration"</screen>
+    </listitem>
+    <listitem>
+     <para>
+      Run keystone-reconfigure Ansible playbook.
+     </para>
+<screen>cd ~/scratch/ansible/next/ardana/ansible
+ansible-playbook -i hosts/verb_hosts keystone-reconfigure.yml</screen>
+    </listitem>
+   </orderedlist>
+  </section>
+  <section>
+   <title>Configure Horizon</title>
+  <para>
+   Complete the following steps to configure horizon to support WebSSO with OpenID method.
+  </para>
+  <procedure>
+   <step>
+    <para>
+     Edit the
+     <literal>~/openstack/ardana/ansible/roles/HZN-WEB/defaults/main.yml</literal>
+     file and set the following parameter to <literal>True</literal>.
+    </para>
+<screen>horizon_websso_enabled: True</screen>
+   </step>
+   <step>
+    <para>
+     Locate the last line in the
+     <literal>~/openstack/ardana/ansible/roles/HZN-WEB/defaults/main.yml</literal>
+     file. The default configuration for this line should look like the
+     following.
+    </para>
+<screen>horizon_websso_choices:
+  - {protocol: saml2, description: "ADFS Credentials"}</screen>
+    <itemizedlist>
+     <listitem>
+      <para>
+       If your cloud <emphasis role="bold">does not</emphasis> have AD FS
+       enabled, then replace the preceding
+       <literal>horizon_websso_choices:</literal> parameter with the following.
+      </para>
+<screen>- {protocol: openid, description: "OpenID Connect"}</screen>
+      <para>
+       The resulting block should look like the following.
+      </para>
+<screen>horizon_websso_choices:
+    - {protocol: openid, description: "OpenID Connect"}</screen>
+     </listitem>
+     <listitem>
+      <para>
+       If your cloud <emphasis role="bold">does</emphasis> have AD FS enabled,
```

Comment:
AD FS or ADFS? Should be consistent throughout.

Reply:
Should be ADFS.